torchgeo/datasets/naip.py

```diff
@@ -42,3 +47,37 @@ class NAIP(RasterDataset):
     # Plotting
     all_bands = ["R", "G", "B", "NIR"]
     rgb_bands = ["R", "G", "B"]
+
+    def plot(  # type: ignore[override]
+        self,
+        sample: Dict[str, Tensor],
```

Comment:
```suggestion
        sample: Dict[str, Any],
```
See my comment on your other PR